libs/cli/langchain_cli/project_template/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11-slim
+
+RUN pip install poetry==1.5.1
```

Comment:
Let's do 1.6.1 to match other libs (can confirm that's right number)

Reply:
fixed